distsql/select_result.go

```diff
@@ -151,6 +158,9 @@ func (r *selectResult) getSelectResp() error {
 			r.selectResp = nil
 			return nil
 		}
+		if r.memTracker != nil {
+			r.memTracker.Consume(-int64(re.result.MemSize()))
```

Comment:
we should also track the memory consumption of `r.selectResp` and `r.feedback`.

Reply:
All other comments have been addressed.
Memory consumption of `r.selectResp` has been traced.
For `QueryFeedback`, it is a bit complex and contains many kinds of structs like `Histogram`, `feedback`, `Chunk`, so we need a fast and graceful way to calculate its memory usage.
Since this PR is big enough, I will do it in another new PR.

Reply:
okay